src/builtins/amp-layout/amp-layout.js

```diff
@@ -19,6 +20,9 @@ export class AmpLayout extends BaseElement {
 
   /** @override */
   buildCallback() {
+    if (isServerRendered(this.element)) {
```

Comment:
see above comment re: putting this check into `buildDom`

Reply:
where is above comment?